packages/pyinstaller/bitmessagemain.spec

```diff
@@ -46,17 +62,16 @@ def addUIs():
 a.datas += addTranslations()
 a.datas += addUIs()
 
-if ctypes.sizeof(ctypes.c_voidp) == 4:
-	arch=32
-else:
-	arch=64
+
 
 a.binaries += [('libeay32.dll', openSSLPath + 'libeay32.dll', 'BINARY'),
-	(os.path.join('bitmsghash', 'bitmsghash%i.dll' % (arch)), os.path.join(srcPath, 'bitmsghash', 'bitmsghash%i.dll' % (arch)), 'BINARY'),
-	(os.path.join('bitmsghash', 'bitmsghash.cl'), os.path.join(srcPath, 'bitmsghash', 'bitmsghash.cl'), 'BINARY'),
-	(os.path.join('sslkeys', 'cert.pem'), os.path.join(srcPath, 'sslkeys', 'cert.pem'), 'BINARY'),
-	(os.path.join('sslkeys', 'key.pem'), os.path.join(srcPath, 'sslkeys', 'key.pem'), 'BINARY')
-	]
+         ('python27.dll', pythonDllPath + 'python27.dll', 'BINARY'),
+         ('msvcr120.dll', msvcrDllPath + 'msvcr120.dll','BINARY'),
+         (os.path.join('bitmsghash', 'bitmsghash%i.dll' % (arch)), os.path.join(srcPath, 'bitmsghash', 'bitmsghash%i.dll' % (arch)), 'BINARY'),
+         (os.path.join('bitmsghash', 'bitmsghash.cl'), os.path.join(srcPath, 'bitmsghash', 'bitmsghash.cl'), 'BINARY'),
+         (os.path.join('sslkeys', 'cert.pem'), os.path.join(srcPath, 'sslkeys', 'cert.pem'), 'BINARY'),
+         (os.path.join('sslkeys', 'key.pem'), os.path.join(srcPath, 'sslkeys', 'key.pem'), 'BINARY')
+         ]
 
 with open(os.path.join(srcPath, 'version.py'), 'rt') as f:
```

Comment:
This should probably be more like `from version.py import softwareVersion`. It's a leftover bug I haven't fixed yet.